pennylane/templates/subroutines/select.py

```diff
@@ -69,10 +105,16 @@ class Select(Operation):
         bitstring representing the control state we can use the following relationship:
         ``index = int(state_string, 2)``. For example, ``2 = int('10', 2)``.
 
+    .. note::
+        Using ``partial=True`` assumes that the quantum state :math:`|\psi\rangle` on the
+        ``control`` wires satisfies :math:`\langle j|\psi\rangle=0` for all :math:`j\in [K, 2^c)`,
+        where :math:`K` is the number of operators (``len(ops)``) and :math:`c` is the number of
+        control wires (``len(control)``).
```

Comment:
link compilation hub page for description of what partial-select is :)

Reply:
Also, shouldnt the keyword args be described in the docstring of Select somewhere?